docx2pdf.py

```diff
@@ -38,20 +38,26 @@ def main_docx2pdf_apres_osmose():
     mkdir_ifnotexist(OUTPUT_DIR)
     mkdir_ifnotexist(DIR_COPY_DOCX)
     mkdir_ifnotexist(avant_osmose_pdf)
-    depname2num = create_dico_dep2name(taxo_dep_df)
+    depname2num = create_dico_dep2num(taxo_dep_df)
     # Mapping docx -> nom pdf
     docx2pdf_filename, doc_odt = docxnames_to_pdfnames(DIR_TO_CONVERT, depname2num)
     check_duclicated_docx(docx2pdf_filename)
     #Archivage a faire ici des docx
     export_to_pdf_apres_osmose(docx2pdf_filename, OUTPUT_DIR, doc_odt, depname2num)
 
 
-def mkdir_ifnotexist(path) :
-    if not os.path.isdir(path) :
+def mkdir_ifnotexist(path: str):
+    """
+    Create a folder if it's doesn't exist
+    """
+    if not os.path.isdir(path):
         os.mkdir(path)
         
 
-def create_dico_dep2name(taxo_dep_df):
+def create_dico_dep2num(taxo_dep_df: pd.DataFrame) -> dict:
+    """
+    Create crossing dictionnary between department and department's number
```

Comment:
Creates crossing dictionnary between department's **name** and department's number